kafka/4.2/modules/ROOT/pages/kafka-connector-reference.adoc

```diff
@@ -0,0 +1,1558 @@
+= Apache Kafka Connector Reference - Mule 4
+:page-aliases: connectors::kafka/kafka-connector-reference.adoc
+ifndef::env-site,env-github[]
```

Comment:
Lines 3, 4, 5 are unnecessary. If you keep them, you need to move the page-aliases line underneath them.